src/ui/layout/tutorial.md

```diff
@@ -244,219 +393,287 @@ class MyApp extends StatelessWidget {
+  const BuildButtonColumn({
+    super.key,
+    required this.color,
+    required this.icon,
+    required this.label,
+  });
+
+  final Color color;
+  final IconData icon;
+  final String label;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+// ···
+    );
+  }
+}
 ```
 
-## Step 4: Implement the text section
+### Update the app to display the button section
+
+Add the button section to the `children` array.
+
+    <?code-excerpt path-base="layout/lakes"?>
+    
+    <?code-excerpt "step{2,3}/lib/main.dart (addWidget)" title?>
+    ```diff
+    --- step2/lib/main.dart (addWidget)
+    +++ step3/lib/main.dart (addWidget)
+    @@ -5,6 +5,7 @@
+             name: 'Oeschinen Lake Campground',
+             location: 'Kandersteg, Switzerland',
+           ),
+    +      ButtonSection(),
+         ],
+       ),
+     ),
+    ```
+
+## Add the Text section
+
+In this section, add the text description to this app.
+
+{% include docs/app-figure.md
+   image="ui/layout/layout-sketch-add-text-block.svg"
+   caption="The text block as sketch and prototype UI" %}
 
-<?code-excerpt path-base="layout/lakes/step4"?>
+<?code-excerpt path-base="layout/lakes"?>
+
+### Add the `TextSection` widget
 
-Define the text section as a variable. Put the text
-in a `Container` and add padding along each edge.
-Add the following code just below the `buttonSection`
-declaration:
+Add the following code as a separate widget after the `ButtonSection` widget.
 
-<?code-excerpt "lib/main.dart (textSection)" title?>
+<?code-excerpt "step4/lib/main.dart (TextSection)" title?>
 ```dart
-Widget textSection = Container(
-  padding: const EdgeInsets.all(32),
-  child: const Text(
-    'Lake Oeschinen lies at the foot of the Blüemlisalp in the Bernese '
-    'Alps. Situated 1,578 meters above sea level, it is one of the '
-    'larger Alpine Lakes. A gondola ride from Kandersteg, followed by a '
-    'half-hour walk through pastures and pine forest, leads you to the '
-    'lake, which warms to 20 degrees Celsius in the summer. Activities '
-    'enjoyed here include rowing, and riding the summer toboggan run.',
-    softWrap: true,
-  ),
-);
+class TextSection extends StatelessWidget {
+  const TextSection({
+    super.key,
+    required this.description,
+  });
+
+  final String description;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(32),
+      child: Text(
+        description,
+        softWrap: true,
+      ),
+    );
+  }
+}
 ```
 
-By setting `softwrap` to true, text lines will fill the column width before
+By setting `softwrap` to `true`, text lines fill the column width before
 wrapping at a word boundary.
 
-Add the text section to the body:
+### Update the app to display the text section
 
-<?code-excerpt path-base="layout/lakes"?>
-<?code-excerpt "{step3,step4}/lib/main.dart" from="return MaterialApp"?>
+Insert the `TextSection` call into the `body` property.
```

Comment:
Creating a new instance of a class through a constructor is not really a "call" like a method, but rather a creation or construction. Also I didn't really know what `body` was referring to at first, could we be more specific about where in the body and say something like:

```suggestion
Add a new `TextSection` widget as a child after the `ButtonSection`.
```